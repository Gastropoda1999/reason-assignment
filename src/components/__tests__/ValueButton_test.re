open Jest;
open Expect;

let _ =

describe("ValueButton", () => {
  let dispatch = _action => ();
  let value = "1";

  test("renders", () => {
    let tree = <ValueButton dispatch value />
      |> ReactShallowRenderer.renderWithRenderer;

    expect(tree) |> toMatchSnapshot;
  });

  test("renders expanded button", () => {
    let tree = <ValueButton dispatch expand=true value />
      |> ReactShallowRenderer.renderWithRenderer;

    expect(tree) |> toMatchSnapshot;
  });
});
