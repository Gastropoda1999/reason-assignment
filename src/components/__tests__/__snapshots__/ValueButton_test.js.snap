// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValueButton renders 1`] = `
<button
  className="css-1csotgl css-1efi2jj css-tdosd6"
  onClick={[Function]}
  value="1"
>
  1
</button>
`;

exports[`ValueButton renders expanded button 1`] = `
<button
  className="css-1csotgl css-1efi2jj css-tdosd6 css-94frz2"
  onClick={[Function]}
  value="1"
>
  1
</button>
`;
